test/sequential/test-fs-watch.js

```diff
@@ -126,19 +126,3 @@ tmpdir.refresh();
   });
   oldhandle.close(); // clean up
 }
-
-{
-  let oldhandle;
-  assert.throws(() => {
-    const w = fs.watchFile(__filename,
-                           { persistent: false },
-                           common.mustNotCall());
-    oldhandle = w._handle;
-    w._handle = { stop: w._handle.stop };
-    w.stop();
-  }, {
-    message: 'handle must be a StatWatcher',
-    code: 'ERR_ASSERTION'
-  });
-  oldhandle.stop(); // clean up
-}
```

Comment:
Does the removal of this test case mean that we have to make the change semver-major? A lot of projects out there use the “tested = supported” rule.

Reply:
@TimothyGu The reason for this change is that the `stop` method on the handle itself is gone now. You could consider this a breaking change, but everything that has changed is two layers of undocumented internals deep.

Also, the reason for this particular test here was to make sure that the native `stop` method would not crash with a segfault in the case of an invalid receiver, not for checking behaviour of the watcher API itself.